ci/github-script/merge.js

```diff
@@ -143,6 +151,7 @@ async function handleMerge({
     return members[team_slug]
   }
   const committers = await getTeamMembers('nixpkgs-committers')
+  const trusted_maintainers = await getTeamMembers('nixpkgs-trusted-maintainers')
```

Comment:
This team (obviously) needs to exist before this PR is merged:

https://github.com/orgs/NixOS/teams/nixpkgs-trusted-maintainers